lightning/telephone_example/make_call.py

```diff
@@ -0,0 +1,33 @@
+import vonage
```

Comment:
if its for vonage, then keep it in another folder - named vonage and plivo stuff keep in plivo folder